shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs

```diff
@@ -1442,3 +1444,59 @@ fn test_cassandra_peers_rewrite() {
         );
     }
 }
+
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn test_cassandra_request_throttling() {
+    let _docker_compose =
+        DockerCompose::new("example-configs/cassandra-passthrough/docker-compose.yml");
+
+    let shotover_manager =
+        ShotoverManager::from_topology_file("tests/test-configs/cassandra-request-throttling.yaml");
+
+    let connection = shotover_manager.cassandra_connection("127.0.0.1", 9042);
+    std::thread::sleep(std::time::Duration::from_secs(1));
```

Comment:
what is this sleep for?

Reply:
When the client connects it sends a bunch of requests to gather information about the cluster, we need to make sure we don't trigger the rate limiter with the second connection.

Reply:
Oh nice. That should definitely be added as a comment.

Reply:
It's a comment underneath, but I'll add it the first instance of sleep too.